internal/secrets/resolver_test.go

```diff
@@ -67,14 +68,21 @@ var _ = Describe("Secrets Resolver", func() {
 	}
 
 	validateSecretResolved := func() {
-		resolvedSecret, err := resolver.GetSecretForResource(ctx, testNamespace, secrets.SAPBTPOperatorSecretName)
+		resolvedSecret, err := resolver.GetSecretForResource(ctx, testNamespace, secrets.SAPBTPOperatorSecretName, "")
+		Expect(err).ToNot(HaveOccurred())
+		Expect(resolvedSecret).ToNot(BeNil())
+		Expect(string(resolvedSecret.Data["clientid"])).To(Equal(expectedClientID))
+	}
+
+	validateSubaccountSecretResolved := func(subaccountID string) {
```

Comment:
if it is used only once and for a specific test the method is not needed, put the code in the test it is more readable

Reply:
i thought it can be reused in the future

Reply:
but i applied your comment